jax/_src/third_party/scipy/special.py

```diff
@@ -275,13 +275,41 @@ def fresnel(x: ArrayLike) -> tuple[Array, Array]:
 
   return s, c
 
+
+@jit
+def sincospisquaredhalf(
+  x: Array,
+) -> tuple[Array, Array]:
+  """
+  Accurate evaluation of sin(pi * x**2 / 2) and cos(pi * x**2 / 2).
+  """
+  x = jnp.abs(x)
+  # x*x/2 mod 2 = ((x mod 2) * (x/2 mod 2)) mod 2
```

Comment:
This formula is true for integers, but not in general true for real numbers.

Reply:
Yes good point, thanks!